api/src/main/java/org/openmrs/module/atomfeed/client/AtomFeedClientFactory.java

```diff
@@ -0,0 +1,27 @@
+package org.openmrs.module.atomfeed.client;
+
+import org.ict4h.atomfeed.client.service.EventWorker;
+import org.openmrs.module.atomfeed.client.impl.AtomFeedClientImpl;
+
+public class AtomFeedClientFactory {
+
+    private final static int SECOND = 1000;
+    private final static int CONNECT_TIMEOUT = 15 * SECOND;
+    private final static int READ_TIMEOUT = 30 * SECOND;
+
+    private AtomFeedClientFactory() { }
+
+    public static AtomFeedClient createClient() {
```

Comment:
Why do we need this method? I can't see any use of it in code. Is it for testing purposes?

Reply:
I wanted the factory to have same constructors as AtomFeedClientImpl.

Reply:
So if it isn't used anywhere I think that we should remove it.